htdocs/admin/mails_templates.php

```diff
@@ -254,7 +254,7 @@
 	$action = 'list';
 	$massaction = '';
 }
-if (!GETPOST('confirmmassaction', 'alpha') && $massaction != 'presend' && $massaction != 'confirm_presend') {
+if (!GETPOST('confirmmassaction', 'alpha') && !empty($massaction) && $massaction != 'presend' && $massaction != 'confirm_presend') {
```

Comment:
This has a different result.
If massaction is not set, test is now false. Before it was true.